l5kit/l5kit/tests/geometry/transform_test.py

```diff
@@ -2,7 +2,22 @@
 import pytest
 import transforms3d
 
-from l5kit.geometry import transform_point, transform_points
+from l5kit.geometry import transform_point, transform_points, transform_points_batch
+
+
+def test_transform_batch_points() -> None:
+    # transform batch and singular elements one by one, results should match
+    # note: we use random here as the validity of transform is checked below
+
+    tfs = np.random.randn(16, 3, 3)
```

Comment:
can you also test if we can jus pass in transform matrix of 2D?

Reply:
> transform matrix of 2D

Do you mean a single matrix for a batch of sequences?

Reply:
yes, boardcasting basically